sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -63,6 +63,11 @@ export interface CorrelationFilter {
     userProperties?: any;
 }
 
+// @public
+export interface CreateBatchOptions {
```

Comment:
Shouldn't this extend OperationOptions?

Reply:
I think it should, added now.
We haven't covered the implementation part related to the abortSignal or retryOptions in the methods or even passing the options internally. 
We have the issues logged - https://github.com/Azure/azure-sdk-for-js/issues/4375 https://github.com/Azure/azure-sdk-for-js/issues/4376